src/test/java/gmail/taras/shevchuk0024/lesson37_Collections/NumberUtilsTest.java

```diff
@@ -0,0 +1,32 @@
+package gmail.taras.shevchuk0024.lesson37_Collections;
+
+import org.junit.jupiter.api.Test;
+
+import java.util.Arrays;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class NumberUtilsTest {
+
+    @Test
+    void shouldReturnCorrectSumOfNumbers() {
+        // given
+        List<Integer> numbers = Arrays.asList(11, 22, 33, 44, 55, 66, 77, 88, 99);
```

Comment:
Краще використати `List.of`.
```suggestion
        List<Integer> numbers = List.of(11, 22, 33, 44, 55, 66, 77, 88, 99);
```